libledger/Ledger.cpp

```diff
@@ -133,9 +134,18 @@ void Ledger::initIniConfig(std::string const& iniConfigFileName)
 
 void Ledger::initTxPoolConfig(ptree const& pt)
 {
-    m_param->mutableTxPoolParam().txPoolLimit = pt.get<uint64_t>("tx_pool.limit", 102400);
-    Ledger_LOG(DEBUG) << LOG_BADGE("initTxPoolConfig")
-                      << LOG_KV("limit", m_param->mutableTxPoolParam().txPoolLimit);
+    try
+    {
+        m_param->mutableTxPoolParam().txPoolLimit =
+            pt.get<uint64_t>("tx_pool.limit", SYNC_TX_POOL_SIZE_DEFAULT);
+        Ledger_LOG(DEBUG) << LOG_BADGE("initTxPoolConfig")
+                          << LOG_KV("txPoolLimit", m_param->mutableTxPoolParam().txPoolLimit);
+    }
+    catch (std::exception& e)
+    {
+        m_param->mutableTxPoolParam().txPoolLimit = SYNC_TX_POOL_SIZE_DEFAULT;

```

Comment:
Is it possible that `m_param->mutableTxPoolParam().txPoolLimit = pt.get<uint64_t>("tx_pool.limit", SYNC_TX_POOL_SIZE_DEFAULT);`  will throw exception?